tests/nvme/rc

```diff
@@ -264,11 +376,16 @@ _create_nvmet_port() {
 		fi
 	done
 
+	if [[ "${adrfam}" = "fc" ]]; then
+		trsvcid=
+	fi
 	mkdir "${NVMET_CFS}/ports/${port}"
 	echo "${trtype}" > "${NVMET_CFS}/ports/${port}/addr_trtype"
 	echo "${traddr}" > "${NVMET_CFS}/ports/${port}/addr_traddr"
 	echo "${adrfam}" > "${NVMET_CFS}/ports/${port}/addr_adrfam"
-	echo "${trsvcid}" > "${NVMET_CFS}/ports/${port}/addr_trsvcid"
+	if [[ "${trsvcid}" ]]; then
```

Comment:
If we change this condition check as follows, previous hunk is not required, I think:

```
if [[ "${adfram}" != "fc" ]]; then
```